go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -786,8 +786,10 @@ func (throttler *Throttler) collectMySQLMetrics(ctx context.Context, tmClient tm
 
 					var throttleMetricFunc func() *mysql.MySQLThrottleMetric
 					if clusterName == selfStoreName {
+						// Throttler probing its own tablet's metrics
```

Comment:
Nit, but IMO we should consistently use sentence capitalization and punctuation in comments.

Reply:
fixed